src/Stratis.Bitcoin.IntegrationTests/EnvironmentMockUpHelpers/NodeRunners.cs

```diff
@@ -43,16 +46,17 @@ public void Kill()
             }
         }
 
-        public void Start(string dataDir)
+        public void Start(string dataDir, bool mineCoinsFast)
         {
-            this.process = Process.Start(new FileInfo(this.bitcoinD).FullName,
-                $"-conf=bitcoin.conf -datadir={dataDir} -debug=net");
+            this.process = Process.Start(new FileInfo(this.bitcoinD).FullName, $"-conf=bitcoin.conf -datadir={dataDir} -debug=net");
         }
     }
 
     public class StratisBitcoinPosRunner : INodeRunner
```

Comment:
Notice a lot of duplicate code in NodeRunner.cs.  At some point good to pull out the common code into a base class.  Something like this for staking logic.

![image](https://user-images.githubusercontent.com/16000683/38772296-0c80d6e2-402b-11e8-8983-89fee1533823.png)

Reply:
Yeah we will do this soon!